cmd/livepeer/livepeer.go

```diff
@@ -391,8 +391,11 @@ func main() {
 			n.Recipient.Start()
 			defer n.Recipient.Stop()
 
+			n.Balances = core.NewBalances(smCleanupInterval)
```

Comment:
If we're going to be using the value of `smCleanupInterval` in multiple places, mind changing the variable name to make it clear that it isn't specifically for `pm.SenderMonitor`? Ex. `cleanupInterval`

Reply:
fixed in https://github.com/livepeer/go-livepeer/pull/956/commits/789195926a3d4628b3bc3e6acec20f4b78035e68